test/E2ETests/E2EApps/E2EApp/E2EApp.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
     
   <PropertyGroup Condition="$(FunctionsRuntimeVersion) == '3'">
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <AzureFunctionsVersion>v3</AzureFunctionsVersion>
```

Comment:
Oh this is still v3. Should we switch to v4?

Reply:
Yes. I'll update this